scala/README.md

```diff
@@ -382,21 +441,41 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
 
     1. Instead of `c exists (_ == 0)`, use `c contains 0`.
 
-0. Do not import `scala.collection.mutable._` or even a single mutable collection directly. Instead, import the `mutable` package and use it explicitly as a namespace prefix to denote mutability:
+    This is not only a matter of style: for some collections, the recommendations above can be algorithmically more efficient than the naive approach.
+
+0. Do not import `scala.collection.mutable._` or even any single mutable collection directly. Instead, import the `mutable` package itself and use it explicitly as a namespace prefix to denote mutability, which also avoids name conflicts if using both mutable and immutable structures in the same scope:
```

Comment:
ok, i thought it did, but maybe i read it somewhere else. i’ve seen this
mentioned in several places though.

On Tue, Jul 28, 2015 at 5:14 PM, Marconi Lanna <notifications@github.com>
wrote:

> In scala/README.md
> <https://github.com/Originate/guide/pull/3#discussion_r35702046>:
>
> > @@ -382,21 +441,41 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
> >
> >      1. Instead of `c exists (_ == 0)`, use `c contains 0`.
> >
> > -0. Do not import `scala.collection.mutable._` or even a single mutable collection directly. Instead, import the `mutable` package and use it explicitly as a namespace prefix to denote mutability:
> > +    This is not only a matter of style: for some collections, the recommendations above can be algorithmically more efficient than the naive approach.
> > +
> > +0. Do not import `scala.collection.mutable._` or even any single mutable collection directly. Instead, import the `mutable` package itself and use it explicitly as a namespace prefix to denote mutability, which also avoids name conflicts if using both mutable and immutable structures in the same scope:
>
> @matthandlersux <https://github.com/matthandlersux> no, i don't think the
> oficial guide mentions it, but it is generally considered best practice.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Originate/guide/pull/3/files#r35702046>.
>



-- 

*Matt Handler*Director of SoftwareOriginate - New York